adot/utils/soak/soak.py

```diff
@@ -93,8 +93,8 @@ def parse_args():
             "soak time:\t%s sec\n"
             "invoke interval:\t%s sec\n"
             "alarm cpu threshold:\t%s ns\n"
-            "alarm memory threshold:\t%s%%"
-            "architecture:\t%s%%"
+            "alarm memory threshold:\t%s % \n"
+            "architecture:\t%s"
```

Comment:
Is this correct?  Looks like a `printf`-style format string, which requires `%%` to emit a literal `%`.